rpm/libpcap.spec

```diff
@@ -0,0 +1,77 @@
+Name: libpcap
+Version: 1.5.3
+Release: 1
+Summary: A system-independent interface for user-level packet capture
+Group: Development/Libraries
+License: BSD with advertising
+URL: http://www.tcpdump.org
+BuildRequires: bison
+BuildRequires: flex
+
+Source: %{name}-%{version}.tar.gz
+
+%description
+Libpcap provides a portable framework for low-level network
+monitoring.  Libpcap can provide network statistics collection,
+security monitoring and network debugging.  Since almost every system
+vendor provides a different interface for packet capture, the libpcap
+authors created this system-independent API to ease in porting and to
+alleviate the need for several system-dependent packet capture modules
+in each application.
+
+Install libpcap if you need to do low-level network traffic monitoring
+on your network.
+
+%package devel
+Summary: Libraries and header files for the libpcap library
+Group: Development/Libraries
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+Libpcap provides a portable framework for low-level network
+monitoring.  Libpcap can provide network statistics collection,
+security monitoring and network debugging.  Since almost every system
+vendor provides a different interface for packet capture, the libpcap
+authors created this system-independent API to ease in porting and to
+alleviate the need for several system-dependent packet capture modules
+in each application.
+
+This package provides the libraries, include files, and other
+resources needed for developing libpcap applications.
+
+%prep
+%setup -q -n %{name}-%{version}/%{name}
+
+#sparc needs -fPIC
+%ifarch %{sparc}
+sed -i -e 's|-fpic|-fPIC|g' configure
+%endif
+
+%build
+export CFLAGS="$RPM_OPT_FLAGS -fno-strict-aliasing"
+%configure
+make %{?_smp_mflags}
+
+%install
+make DESTDIR=$RPM_BUILD_ROOT install
+rm -f $RPM_BUILD_ROOT%{_libdir}/libpcap.a
+
+%post -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%files
+%defattr(-,root,root)
+%doc LICENSE README CHANGES CREDITS
+%{_libdir}/libpcap.so.*
+%{_mandir}/man7/pcap*.7*
```

Comment:
Man page also could go to devel (or actually -docs or something but not sure if we have proper policy for that for now), we don't need it on device.